packages/native-ui/src/config/theme/helpers/Button.ts

```diff
@@ -8,50 +8,53 @@ export const invertedOutlineScheme = (scheme: Scheme) => ({
   colorScheme: scheme,
   variant: 'outline',
   value: {
+    borderColor: `$${scheme}400`,
+
     _text: {
-      color: `$${scheme}50`,
+      color: `$${scheme}100`,
     },
 
     _icon: {
-      color: `$${scheme}50`,
+      color: `$${scheme}100`,
     },
 
     _spinner: {
       _svg: {
-        color: `$${scheme}50`,
+        color: `$${scheme}100`,
       },
     },
 
     ':active': {
       backgroundColor: `$${scheme}900`,
+      borderColor: `$${scheme}400`,
 
       _text: {
-        color: `$${scheme}50`,
+        color: `$${scheme}100`,
       },
 
       _icon: {
-        color: `$${scheme}50`,
+        color: `$${scheme}100`,
       },
 
       _spinner: {
         _svg: {
-          color: `$${scheme}50`,
+          color: `$${scheme}100`,
         },
       },
     },
 
     ':disabled': {
-      borderColor: `$${scheme}700`,
+      borderColor: `$${scheme}600`,
 
       _text: {
-        color: `$${scheme}700`,
+        color: `$${scheme}600`,
```

Comment:
question here: does this apply to ghost too?

Reply:
<img width="1718" alt="Screenshot 2024-08-08 at 16 06 39" src="https://github.com/user-attachments/assets/7dc04f03-b1d1-4b2a-acae-d2d72b304a6b">


the `disabled` on `inverted` Button `variant="ghost"` was changed from 700 to 600

<img width="516" alt="Screenshot 2024-08-08 at 16 09 09" src="https://github.com/user-attachments/assets/b26fddd5-a110-4b4c-b5ab-7cf6d93cdba7">

Reply:
Cheers, missed that one